util/winutil/winutil.go

```diff
@@ -13,45 +13,45 @@ import (
 const RegBase = regBase
 
 // GetPolicyString looks up a registry value in the local machine's path for
-// system policies, or returns the given default if it can't.
+// system policies, or returns empty string and the error.
 // Use this function to read values that may be set by sysadmins via the MSI
 // installer or via GPO. For registry settings that you do *not* want to be
 // visible to sysadmin tools, use GetRegString instead.
 //
 // This function will only work on GOOS=windows. Trying to run it on any other
-// OS will always return the default value.
-func GetPolicyString(name, defval string) string {
-	return getPolicyString(name, defval)
+// OS will always return an empty string.
```

Comment:
I feel like it should return an error on non-Windows, but I guess this is not important and it's going to be replaced promptly anyway.

Reply:
Especially now that the default is gone, we need to document what the error value is on a miss. Is it an error for a requested registry key to not exist or does it return the zero value? Document.